students/ZackConnaughton/session04/trigrams.py

```diff
@@ -0,0 +1,73 @@
+#!/usr/bin/env python
+
+"""
+trigrams!
+"""
+import random
+
+def strip_punct(s):
+    output = s
+    if '--' in output:
```

Comment:
you don't need to check first -- if you call replace, and it's not there, it won't do anything.